### PR TITLE
Docs: Add FAQ: How to update

### PR DESCRIPTION
**Pull request by jaredbeck**
I couldn't find any docs on how to update, so I thought I'd try to write some. What I've written is just my best guess, improvements welcome.

### Motivation

I'm trying to upgrade from 0.4.4314 to 0.4.4358, getting errors I don't understand, and can't find any instructions.

### Test plan

Whatever is standard for docs?

### CODE REVIEW

File: website/docs/faq.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ title: Frequently Asked Questions
...
+
+Sorbet has not reached version 1.0 yet. As such, it will make 
+breaking changes constantly, without warning.
+
+To upgrade a patch level, eg. from 0.4.4314 to 0.4.4358,
+
+```
+bundle update sorbet sorbet-runtime 
+# also update plugins like sorbet-rails, if any
+rm -r sorbet
```
Comment (jez) [inline website/docs/faq.md]: Not needed, `srb init` will regenerate the `sorbet/` folder.

File: website/docs/faq.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ title: Frequently Asked Questions
...
+bundle exec sorbet init
```
Comment (jez) [inline website/docs/faq.md]: What is `sorbet init`? Did you mean `srb init`?

```suggestion
bundle exec srb init
```

**Comment by jaredbeck** [comment]
Corrections made, thanks!

**Comment by jez** [comment]
@jaredbeck We use Prettier for Markdown; I checked out your branch and fixed the prettier errors. In the future, feel free to clone your branch locally and fix them yourself to get a faster turn around.

Also, I moved the question you added further down on the page (because the ones that are already there are more frequently asked!).

Regardless, I'll merge when the tests pass. Thanks again!

**Pull request merged**